.yarn/patches/debug-npm-4.3.4-4513954577.patch

```diff
@@ -0,0 +1,70 @@
+diff --git a/src/browser.js b/src/browser.js
+index cd0fc35d1ee11e0d6e15421021a54c18958e04d9..ac3f7e1339b9852e42b502eb0c049a32c455c5b0 100644
+--- a/src/browser.js
++++ b/src/browser.js
+@@ -9,16 +9,6 @@ exports.save = save;
+ exports.load = load;
+ exports.useColors = useColors;
+ exports.storage = localstorage();
+-exports.destroy = (() => {
```

Comment:
We don't want to completely remove it, just make the function a `noop` (no op)eration) 
Because somewhere in our code (or a dependency library) it's being called